sql/parser/sql.y

```diff
@@ -336,7 +338,7 @@ import "strings"
 %token <str>   FALSE FAMILY FETCH FILTER FIRST FLOAT FOLLOWING FOR
 %token <str>   FORCE FOREIGN FORWARD FREEZE FROM FULL FUNCTION FUNCTIONS
 
-%token <str>   GLOBAL GRANT GRANTED GREATEST GROUP GROUPING
+%token <str>   GLOBAL GRANT GRANTED GRANTS GREATEST GROUP GROUPING
```

Comment:
I think `GRANTS` need to be added to the `unreserved_keywords` rule. Otherwise, it can't be used as a `name`.